wallet/src/de/schildbach/wallet/ui/main/WalletActivity.java

```diff
@@ -148,14 +147,6 @@ protected void onResume() {
         checkLowStorageAlert();
         checkWalletEncryptionDialog();
         detectUserCountry();
-        showBackupWalletDialogIfNeeded();
-    }
-
-    private void showBackupWalletDialogIfNeeded() {
-        if (showBackupWalletDialog) {
-            BackupWalletDialogFragment.show(getSupportFragmentManager());
-            showBackupWalletDialog = false;
-        }
     }
```

Comment:
Looks like we don't need this anymore. `showBackupWalletDialog` is never set to `true`.

Reply:
Correct.